portfolio/src/main/webapp/script.js

```diff
@@ -31,8 +31,12 @@ function deleteComments() {
   fetch("/delete-data", {method: 'post'}).then(getComments());
 }
 
-function createListElement(text) {
+function createListElement(message, score) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  liElement.innerText = message + ' (' + decideMorality(score) + ')';
   return liElement;
+}
+
+function decideMorality(score) {
```

Comment:
add a comment explaining what's the possible outcome moralities and how the decision is made (i.e. what does it mean to have a negative score?)